Pipelines/Templates/set-environment-id.yml

```diff
@@ -4,18 +4,12 @@ parameters:
 
 steps:
 - powershell: |    
-    Import-Module Microsoft.PowerApps.Administration.PowerShell -Force -RequiredVersion $(PowerAppsAdminModuleVersion) -ArgumentList @{ NonInteractive = $true }
-    Add-PowerAppsAccount -TenantID $(TenantId) -ApplicationId $(ClientId) -ClientSecret $(ClientSecret)
-    Import-Module Microsoft.Xrm.Data.PowerShell -Force -RequiredVersion $(XrmDataPowerShellVersion) -ArgumentList @{ NonInteractive = $true }
-    $conn = Get-CrmConnection -ConnectionString "$(CdsBaseConnectionString)${{parameters.serviceConnection}}"
-
-    # Get the EnvironmentName (which is a GUID) of the environment based on the orgid in Dataverse
-    $orgId = (Get-CrmRecords -conn $conn -EntityLogicalName organization).CrmRecords[0].organizationid
-
-    # Get the Environment using the InstanceId
-    $environment = Get-AdminPowerAppEnvironment -InstanceId $orgId
+    Install-Module -Name Microsoft.Xrm.Tooling.CrmConnector.PowerShell -Force #-RequiredVersion $(PowerAppsAdminModuleVersion)

```

Comment:
In this line I have commented out the RequiredVersion for the cmdlet, although we might want to consider creating a new variable for XRM Tooling unless there is one already I'm not aware of.

The last time the package was updated was 6 months ago, so i don't believe we will see a lot of changes.